### ISSUE TITLE
Simplify switching accounts in nav drawer

### ISSUE THREAD

**Issue by aseigo** [konradrenner/kolabnotes-android#47]
There are multiple ways to switch accounts in the nav drawer: pressing on the drop down next to the account name; pressing on one of the icons that represent other accounts. The icons for accounts are, by default, using the same icon ... and if I were to set them up to something unique, they'd probably end up with my little gravitar icon, and so look the same once again. When the icons look the same, it becomes a guessing game as to which account I will switch to.

IMHO only one way to switch accounts is really necessary, and given the ambiguity of the icon accounts, perhaps move just to the drop down list? (Icons could then be folded into that representation). This would have the nice side effect of making it possible to make the drawer header smaller.

Alternatively, it might be desirable to look for gravitar icons, use the Android logo or a devie icon for the default Local collection (or something else that says "this is on your device"), check if there is a favicon available related to the service (generally checking for webmail.<service domain> or <service domain>/apps will work) ...

**Comment by konradrenner** [comment]
In principle the nav drawer behaves like nav drawers from Google apps (e.g. GMail, Keep), they also have the dropdown and the icons are selectable. I have not implemented the drawer on my own, I use the Material drawer lib from Mike Penz (btw thank you for creating such a lib as an OSS project :-) ).

But it is a problem at the moment, that one can not specify an icon for an account (it is possible to set icons with the used lib). I have not implemented it yet because I am not sure how to sync the icon (it would be a no-go for me, if I have to specify an icon for my accounts on each device separatly). @aseigo  do you know if it is possible to set an icon for a Kolab account on the web UI and how to get it from the server?

Thank you for the hint, to specify a default icon for the local account :+1:

**Comment by konradrenner** [comment]
@aseigo thank you for your input. I found a way to configure the drawer so that it uses a "minimal" mode. That means account switching is done just via the list. The header is now smaller and the problem with the duplicate account icons is also gone :-). Also I have implemented a check, if the email is the same as the account name, if this is true, just the email is shown.

It will be available with the next version of the app (0.1.0-beta5), which I will release in this week
